### PR TITLE
Add settled state

### PR DESCRIPTION
**Pull request by delfipolito**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "@aragon/connect-agreement": "^0.0.0-beta.4",
```
Comment (andy-hook) [inline package.json:37]: Should we be upgrading agreement as well?

Comment (delfipolito) [reply package.json:37]: I didnt know if it would break something, that's why i didnt do it on this PR

Comment (delfipolito) [reply package.json:37]: But yeah! Should I do it in this one?

Comment (andy-hook) [reply package.json:37]: I'm just going to check a few things, I'm seeing errors on mainnet with this change.

**Review by andy-hook — APPROVED** [review]
We're in business 🙌

**Pull request merged**